### DISCUSSION TITLE
请问一下lantern iOS最新版本，没有找到动态码授权的入口

### DISCUSSION THREAD

**Discussion by jiweichao1986** [getlantern/lantern-forum-cn#265] [General]
在美区下载了最新iOS latern，进入后账号部分没有找到动态码信息，无法在桌面版完成授权，请问一下最新版本iOS哪里能看到授权的动态码

**Comment by AvatarVirgo** [comment]
同问

**Comment by MaxMKMax** [comment]
@jiweichao1986 @AvatarVirgo 如果您是受到授权码问题影响的IOS用户，我们将通过重新延长您的专业版使用时间来补偿您。
现在iOS系统暂时无法登陆. 我们正在弄新版本了, 很快可以更新了. 请您耐心等待. 非常抱歉影响了您的体验感!

**Reply by YuanGA007** [reply]
我刚下载了ios版本，也是无法授权专业版，什么时候修复
